index.html

```diff
@@ -20,7 +20,15 @@
         <ul id="navList">
           <li><a href="#about-me">About me</a></li>
           <li><a href="#projects">Projects</a></li>
-          <li><a href="#contact-me">Contact me</a></li>
+          <li class="social-fan">
+            <a href="#contact-me">Contact me</a>
+            <div class="hidden-pill social email hidden">
```

Comment:
I think this would be better:
```suggestion
            <div class="hidden-pill social hidden">
```